aksel.nav.no/website/pages/eksempler/formsummary/editlink-variation.tsx

```diff
@@ -0,0 +1,49 @@
+import { FormSummary, VStack } from "@navikt/ds-react";
+import { withDsExample } from "@/web/examples/withDsExample";
+
+const Example = () => {
+  return (
+    <VStack gap="8">
+      <FormSummary>
+        <FormSummary.Header>
+          <FormSummary.Heading>Personalia</FormSummary.Heading>
+          <FormSummary.EditLink href="#" aria-label="Endre svar" />
+        </FormSummary.Header>
+
+        <FormSummary.Answers>
+          <FormSummary.Answer>
+            <FormSummary.Label>Navn</FormSummary.Label>
+            <FormSummary.Value>Ola Nordmann</FormSummary.Value>
+          </FormSummary.Answer>
+        </FormSummary.Answers>
+      </FormSummary>
+      <FormSummary>
+        <FormSummary.Header>
+          <FormSummary.Heading>Personal</FormSummary.Heading>
+          <FormSummary.EditLink href="#" aria-label="Uppdatera information">
+            Uppdatera information
+          </FormSummary.EditLink>
```

Comment:
Svensk er kanskje ikke så realistisk, eller? Hva med engelsk?

aria-label bør være ulik fra synlig tekst for at det skal være noe vits. Samme over.